stroom-connectors/src/main/java/stroom/connectors/kafka/StroomKafkaProducerRecord.java

```diff
@@ -0,0 +1,153 @@
+package stroom.connectors.kafka;
```

Comment:
This file is essentially a copy of the one from the Kafka client library. But we can't get one from the real kafka client library without bringing in some specific version. This is a common issue we will have to resolve one way or another if we abstract any other libraries out. (i.e. HDFS)